upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template

```diff
@@ -113,25 +121,24 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: weave-net
-  namespace: kube-system
   labels:
     name: weave-net
     role.kubernetes.io/networking: "1"
+  namespace: kube-system
 spec:
   # Wait 5 seconds to let pod connect before rolling next pod
```

Comment:
Should that comment also be moved with the field itself?

Reply:
Thanks. I will add a note to do it with the next release. I don't think it's ok to do it in this cherry-pick.

Reply:
Ohh, I didn't notice that it's a CP, sorry!

Reply:
np. good catch.